operator/cmd/run/run.go

```diff
@@ -161,7 +158,7 @@ func Command() *cobra.Command {
 			// Always run a pprof server to facilitate debugging.
 			go runPProfServer(ctx, pprofAddr)
 
-			Run(
+			if err := Run(
```

Comment:
Should we change to use RunE function of cobra command struct?
```
	// RunE: Run but returns an error.
	RunE func(cmd *Command, args []string) error
```

Reply:
Done, good suggestion!